src/main/java/shortestpath/pathfinder/VisitedTiles.java

```diff
@@ -0,0 +1,61 @@
+package shortestpath.pathfinder;
+
+import net.runelite.api.coords.WorldPoint;
+
+public class VisitedTiles {
+    private static final int TOTAL_PLANES = 4;
+    private static final int REGION_SIZE = 64;
+
+    // TODO: what is the max number of regions?
+    private static final int REGION_EXTENT_X = 192;
+    private static final int REGION_EXTENT_Y = 256;
+    private final VisitedRegion[] visitedRegions = new VisitedRegion[REGION_EXTENT_X * REGION_EXTENT_Y];
+
+    public boolean get(WorldPoint point) {
+        final int regionIndex = point.getRegionID();
```

Comment:
`point.getRegionID()` assumes that `REGION_EXTENT_X = 256`, so this needs to be a custom calculation

Reply:
Since I trimmed unused rows/columns from `visitedTiles`, I've adjusted the calculation to:
```
private int getRegionIndex(int regionX, int regionY) {
    return (regionX - regionExtents.minX) + (regionY - regionExtents.minY) * widthInclusive;
}
```
Where `regionX` is `X / REGION_SIZE` (and same for Y). The division will be optimized into a bitshift at compile time since `REGION_SIZE` is a static power of two so I left it for clarity.

Any regions that are out-of-bounds will now report as being visited to avoid exploring them any further.